Rakefile

```diff
@@ -36,4 +36,4 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task :default => :test
+task default: [:test, :rubocop]
```

Comment:
I suppose i can get behind 'rake test' as an alternative....

Thoughts about making autocorrct easy? Make rake does rubocop:auto_correct and rake ci just rubocop?

B mobile phone

> On Oct 1, 2015, at 8:00 AM, L. Preston Sego III <notifications@github.com> wrote:
> 
> rake test